src/commands/generate/typeGenerator.ts

```diff
@@ -287,8 +303,8 @@ export async function generateUnion(
     const objectLiteralExpr = variableStatement.getDeclarations()[0].getInitializer() as ObjectLiteralExpression;
     sourceFile.getFunctions().forEach(f => {
         objectLiteralExpr.addPropertyAssignment({
-            initializer: f.getName(),
-            name: f.getName(),
+            initializer: f.getName()!,
+            name: f.getName()!,
```

Comment:
super nit: maybe worth a comment that we can use the bang since we defined them above

Reply:
but also it might be really obvious when looking at the code not in a git diff

Reply:
I can add a check.